cmd/access/node_builder/access_node_builder.go

```diff
@@ -149,25 +152,27 @@ type FlowAccessNodeBuilder struct {
 	*AccessNodeConfig
 
 	// components
-	UnstakedLibP2PNode         *p2p.Node
```

Comment:
removed al UnstakedXXX components since there is only instance of libp2p, network and middleware.